src/test/java/bio/terra/service/filedata/google/firestore/FireStoreCleanProject.java

```diff
@@ -0,0 +1,84 @@
+package bio.terra.service.filedata.google.firestore;
+
+import bio.terra.common.category.Connected;
+import bio.terra.common.fixtures.JsonLoader;
+import bio.terra.service.iam.IamService;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.QueryDocumentSnapshot;
+import com.google.cloud.firestore.QuerySnapshot;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles({"google", "connectedtest"})
+@Category(Connected.class)
+public class FireStoreCleanProject {
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private IamService samService;
+
+    @MockBean
+    private TestConfiguration testConfiguration;
+
+    @MockBean
+    private JsonLoader jsonLoader;
+
+    /*
+     * NOTE: This is not a test. This is a tool to delete all FireStore collections from a project.
+     * It is written as a JUnit test because that is the simplest way for me to package it today.
+     * It only deletes the top-level of documents. Since we don't use documents with sub-collections,
+     * that doesn't matter. Just to be clear - it is not general purpose.
```

Comment:
I like the idea of this tool. Could you make it runnable as a Gradle task instead?

Reply:
Hmm... yes, if I add a new Category so it doesn't get run with any other tests, then I think I could make a task that just runs that one test as an app.

The right way to do it would be to code it in Python (just as soon as I learn that) and have it be standalone.

Or move it to a jadecli command and leave it in java

Reply:
For now, I'll copy it out of the tree and delete it from this PR